workspace/graph-sequencer/test/index.ts

```diff
@@ -0,0 +1,366 @@
+import { graphSequencer } from '../src'
+
+test('base test case0', () => {
+  expect(graphSequencer(new Map([
+    ['a', ['a']],
+    ['b', ['b']],
+    ['c', ['c']],
+  ]
+  ), ['a', 'b', 'c'])).toStrictEqual(
+    {
+      safe: true,
+      chunks: [['a', 'b', 'c']],
+      cycles: [
+        ['a'], ['b'], ['c'],
+      ],
+    }
+  )
+})
+
+test('base test case1', () => {
+  expect(graphSequencer(new Map([
+    ['a', ['b', 'c']],
+    ['b', ['b']],
+    ['c', ['b', 'c']]]
+  ), ['a', 'b', 'c'])).toStrictEqual(
+    {
+      safe: true,
+      chunks: [['b', 'c'], ['a']],
+      cycles: [
+        ['b'], ['c'],
+      ],
+    }
+  )
+})
+
+test('base test case2', () => {
+  expect(graphSequencer(new Map([
+    ['a', ['b', 'c']],
+    ['b', []],
+    ['c', ['b']]]
+  ), ['a', 'b', 'c'])).toStrictEqual(
+    {
+      safe: true,
+      chunks: [['b'], ['c'], ['a']],
+      cycles: [],
+    }
+  )
+})
+
+test('base test case3', () => {
+  expect(graphSequencer(new Map([
+    ['a', ['b', 'c']],
+    ['b', []],
+    ['c', []],
+    ['d', ['a']],
+    ['e', ['a', 'b', 'c']]]
+  ), ['a', 'd', 'e'])).toStrictEqual(
+    {
+      safe: true,
+      chunks: [['a'], ['d', 'e']],
+      cycles: [],
+    }
+  )
+})
+
+test('graph with no dependencies', () => {
+  expect(graphSequencer(new Map([
+    [0, []],
```

Comment:
not sure why you switched to numbers in these tests. They used letters in the past

Reply:
Numbers are more concise than strings, without quotes.